packages/core/lib/Configuration.ts

```diff
@@ -284,7 +284,7 @@ export class Configuration {
             type: "number",
           },
           "evaluation-budget": {
-            alias: ["e"],
+            alias: [],
```

Comment:
Why was this a bug?

Reply:
I accidentally used "e" twice (also for exclude)
So it was overwritten.

Reply:
We probably also only need an alias for the main budget: totalTime

Reply:
Created an issue for this.